appender-core/src/main/java/com/van/logging/LoggingEventCache.java

```diff
@@ -284,6 +281,10 @@ public boolean stop() {
             }
         }
 
+        if (null != cacheMonitor) {
```

Comment:
👍 

Actually this is probably a better place to do this even without the bug since the instance cleaning up its `cacheMonitor` is an implementation detail that `shutDown` should not care about.